backend/inspirehep/mailing/views.py

```diff
@@ -56,20 +57,20 @@ def get_weekly_jobs_rss():
     title = raw_email_entry[b"title"].decode("UTF-8")
     content = raw_email_entry[b"html"].decode("UTF-8")
     timestamp = float(raw_email_entry[b"timestamp"])
-    date = datetime.utcfromtimestamp(timestamp)
-
-    feed = AtomFeed(
-        title="Weekly HEP jobs - INSPIRE", feed_url=request.url, url=request.url_root
-    )
-
-    feed.add(
-        id=str(timestamp),
-        title=title,
-        content=content,
-        content_type="html",
-        author="inspirehep.net",
-        updated=date,
-        published=date,
-    )
-
-    return feed.get_response()
+    date = datetime.utcfromtimestamp(timestamp).replace(tzinfo=pytz.UTC)
```

Comment:
why? You have already UTC there.

Reply:
nop it doesn't have `tzinfo` and `feedgen` is complaining because of that.

Reply:
O. You are right. https://docs.python.org/3.6/library/datetime.html#datetime.datetime.utcfromtimestamp This is weird. 
Cannot you use datetime.fromtimestamp(timestamp, timezone.utc) then?

Reply:
yep, that's same result and seems cleaner.